yb-voyager/src/utils/utils.go

```diff
@@ -444,3 +444,24 @@ func GetFSUtilizationPercentage(path string) (int, error) {
 	percUtilization := 100 - int((stats.Bavail*100)/stats.Blocks)
 	return percUtilization, nil
 }
+
+//read the file and return slice of csv 
+func ReadTableNameListFromFile(filePath string) ([]string, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return []string{}, fmt.Errorf("error opening file %s: %v", filePath, err)
+	}
+	defer file.Close()
+	var list []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
```

Comment:
Trim the `line` before using it further -- else one rogue white-space can break this code.